ens/main.py

```diff
@@ -30,6 +30,22 @@
 )
 
 ENS_MAINNET_ADDR = '0x314159265dD8dbb310642f98f50C066173C1259b'
+ENS_ROPSTEN_ADDR = '0x112234455c3a32fd11230c42e7bccd4a84e02010'
+ENS_RINKEBY_ADDR = '0xe7410170f87102df0055eb195163a03b7f2bff4a'
+ENS_GOERLI_ADDR = '0x112234455c3a32fd11230c42e7bccd4a84e02010'
+
+
+def get_address_for_network(net):
+    version = net.version
+
+    if version == 1:
+        return ENS_MAINNET_ADDR
+    elif version == 3:
+        return ENS_ROPSTEN_ADDR
+    elif version == 4:
+        return ENS_RINKEBY_ADDR
+    elif version == 5:
+        return ENS_GOERLI_ADDR
```

Comment:
This should have a final `else` block that throws an exception in the event that no match is found.

Reply:
Done!